### PR TITLE
Add setup option: back-to-top-text

### PR DESCRIPTION
**Pull request by gabrielalmeida**
`Back to top` is the unique text embedded by the theme I guess. Adding this option allows to just setup it in the desired language.

### CODE REVIEW

**Comment by rafaelrinaldi** [comment]
@gabrielalmeida Yeah, that could be added but not sure if it's wanted/expected @caffo? Are you using the theme and miss this feature?

**Comment by gabrielalmeida** [comment]
Yes! I was writing a non-English/English page and had to modify that. I believe that changing strings on the setup file are a bit easier/better than changing it as hard-coded somewhere in the source, maybe someone has a similar use case.

File: src/scripts/index.js [hunk 1]
```diff
@@ -92,10 +92,11 @@
   function addBackToTop() {
     var link = doc.createElement('a');
     var container = doc.querySelector('body');
+    var text = options['back-to-top-text'] || 'Back to top';
```
Comment (caffo) [inline src/scripts/index.js]: Could you add the default text to `theme.json` and remove the conditional here? that way you don't even need this variable.

**Comment by caffo** [comment]
I like the idea. Just added a comment.

**Comment by gabrielalmeida** [comment]
Done!

**Comment by rafaelrinaldi** [comment]
🚀

**Comment by caffo** [comment]
Thanks @gabrielalmeida!

**Pull request merged**